Add Iotty (#29624)
home-assistant/home-assistant.github.io: source/_integrations/iotty.markdown

---
title: iotty
description: "Instructions on how to connect iotty Smart Devices to Home Assistant."

> ha_release: "0.38"

Reviewer: ```suggestion
ha_release: "2024.8"
```
This is the release when the integration was first introduced. For more info on the header, refer to the documentation on [creating a new integration documentation page](https://developers.home-assistant.io/docs/documenting/create-page)

ha_category: "No score"
ha_iot_class: "Cloud Polling"
ha_quality_scale: no score
ha_config_flow: true
ha_codeowners:
  - '@pburgio'
ha_domain: iotty
---

The iotty family is focused on beautiful aesthetics and includes the Smart Switch for lights and gates, the Smart Shades Switch for blinds and shutters and the Smart Outlet. Install the iotty devices and enjoy your new Smart Home with ease!

> 

Reviewer: ```suggestion

## Supported devices

```

This integration currently supports the following iotty Devices:

- iotty Smart Switch \
  US version
  - https://iottysmarthome.com/products/1-switch-controller?variant=43630747058389
  - https://iottysmarthome.com/products/2-switch-controller?variant=43630751219925
  - https://iottysmarthome.com/products/3-switch-controller?variant=43630760493269
  - https://iottysmarthome.com/products/4-switch-controller?variant=43630774386901 \
  
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082

>   - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli

Reviewer 2: Done

More integrations are coming soon:

- iotty Shades Switch \
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-shades-switch
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-tende-e-tapparelle

- iotty Smart Outlet: \
  EU VERSION:
  - https://iotty.fr/collections/frontpage/products/prise-intelligent-iotty
  - https://iotty.de/collections/frontpage/products/iotty-smart-steckdose

>   - https://iotty.it/collections/frontpage/products/iotty-plus-presa-intelligente

Reviewer 2: Discarded, as this is no longer present in the documentation

ha_release: "2024.8"
ha_category: "No score"
ha_iot_class: "Cloud Polling"
ha_quality_scale: no score
ha_config_flow: true
ha_codeowners:
  - '@pburgio'
ha_domain: iotty
---

The iotty family is focused on beautiful aesthetics and includes the Smart Switch for lights and gates, the Smart Shades Switch for blinds and shutters and the Smart Outlet. Install the iotty devices and enjoy your new Smart Home with ease!

## Supported devices

This integration currently supports the following iotty Devices:

- iotty Smart Switch \
  US version
  - https://iottysmarthome.com/products/1-switch-controller?variant=43630747058389
  - https://iottysmarthome.com/products/2-switch-controller?variant=43630751219925
  - https://iottysmarthome.com/products/3-switch-controller?variant=43630760493269
  - https://iottysmarthome.com/products/4-switch-controller?variant=43630774386901 \
  
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli

> More integrations are coming soon:

Reviewer 2: Discarded, as this is no longer present in the documentation

- iotty Shades Switch \
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-shades-switch
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-tende-e-tapparelle

- iotty Smart Outlet: \
  EU VERSION:
  - https://iotty.fr/collections/frontpage/products/prise-intelligent-iotty
  - https://iotty.de/collections/frontpage/products/iotty-smart-steckdose

>   - https://iotty.it/collections/frontpage/products/iotty-plus-presa-intelligente

Reviewer: ```suggestion
  - https://iotty.it/collections/frontpage/products/iotty-plus-presa-intelligente
  
  {% include integrations/config_flow.md %}
```

---
title: iotty
description: "Instructions on how to connect iotty Smart Devices to Home Assistant."
ha_release: "2024.8"
ha_category: "No score"
ha_iot_class: "Cloud Polling"
ha_quality_scale: no score
ha_config_flow: true
ha_codeowners:
  - '@pburgio'

> ha_domain: iotty

Reviewer: ```suggestion
ha_domain: iotty
ha_platforms:
  - switch
 ha_integration_type: hub
```

---
title: iotty
description: "Instructions on how to connect iotty Smart Devices to Home Assistant."
ha_release: "2024.8"

> ha_category: "No score"

Reviewer: ```suggestion
ha_category: 
  - Switch
```

ha_iot_class: "Cloud Polling"
ha_quality_scale: no score
ha_config_flow: true
ha_codeowners:
  - '@pburgio'
ha_domain: iotty
ha_platforms:
  - switch

>  ha_integration_type: hub

Reviewer: ```suggestion
ha_integration_type: hub
```

ha_integration_type: hub
---

The iotty {% term integration%} lets you integrate iotty devices into Home Assistant. The iotty family includes the smart switch for lights and gates, the smart shades switch for blinds and shutters, and the smart outlet. 

{% include integrations/config_flow.md %}

## Prerequisites

> In order to use this integration you must have a iotty account, and enter its credentials during account pairing.

Reviewer: ```suggestion
In order to use this integration, you must have an iotty account, and enter its credentials during account pairing.
```

> To create a iotty account, you need to get the App from the [App Store](https://apps.apple.com/it/app/iotty-smart-home/id1230937401) or [Play Store](https://play.google.com/store/apps/details?id=com.dynamicait.iotty&hl=en).

Reviewer: ```suggestion
To create an iotty account, you need to get the App from the [App Store](https://apps.apple.com/it/app/iotty-smart-home/id1230937401) or [Play Store](https://play.google.com/store/apps/details?id=com.dynamicait.iotty&hl=en).
```

## Supported devices

> This integration currently supports the following iotty Devices:

Reviewer: ```suggestion
This integration currently supports the following iotty devices:
```

  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli

More integrations are coming soon:

- iotty Shades Switch \
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-shades-switch
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-tende-e-tapparelle

- iotty Smart Outlet: \
  EU VERSION:
  - https://iotty.fr/collections/frontpage/products/prise-intelligent-iotty
  - https://iotty.de/collections/frontpage/products/iotty-smart-steckdose
  - https://iotty.it/collections/frontpage/products/iotty-plus-presa-intelligente

>   

Reviewer: Are there any prerequisites? Anything I need to do or have before I can install the iotty integration, for example, do I need an iotty account and login credentials?

Reviewer 2: Done

> ## Supported Entities

Reviewer: ```suggestion
## Supported entities
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization) also in titles

> Each iotty device gets mapped into one Home Assistant device per gang, each with its own Switch entity.

Reviewer: what do you mean by "per gang"? group?

Reviewer 2: Hi, "gang" is a pretty standard term used when referring to te single switches on a plate, as you can see in this wikipedia article: [Light Switch](https://en.wikipedia.org/wiki/Light_switch).

I didn't want to reuse the word "Device" or "Switch" as it would be using the same word to define itself. Let me know if that is clear, or if you have any other suggestion to replace the word "gang"

Reviewer: Thank you for the explanation. Learned something today.

> The main, unnamed switch entity, controls the lightswitch, turning it on or off the gang related to it.

Reviewer: ```suggestion
The main, unnamed, switch entity controls the light switch, turning it on or off the gang related to it.
```

> The main, unnamed, switch entity controls the light switch, turning it on or off the gang related to it.

Reviewer 3: This sentence has incorrect grammar. I can't make a suggestion since it's not clear to me what meaning is meant.

Reviewer 2: Hej Martin, thanks for the feedback. Since the PR has already been merged, what is the suggested course of action here? Open another PR to fix this? Wait for the next release in which some functionality will be added to my integration?

For what concerns the meaning, what I want to convey is:

1. There is only one entity per device
2. The main (and only) entity of the device is unnamed
3. The switch entity is meant to control a light switch

Reviewer 3: Please open a new PR and adjust the sentence.

---
title: iotty
description: "Instructions on how to connect iotty Smart Devices to Home Assistant."
ha_release: "2024.8"
ha_category: 
  - Switch
ha_iot_class: "Cloud Polling"
ha_quality_scale: no score
ha_config_flow: true
ha_codeowners:
  - '@pburgio'
ha_domain: iotty
ha_platforms:
  - switch
ha_integration_type: hub
---

> The iotty family is focused on beautiful aesthetics and includes the Smart Switch for lights and gates, the Smart Shades Switch for blinds and shutters and the Smart Outlet. Install the iotty devices and enjoy your new Smart Home with ease!

Reviewer: ```suggestion
The iotty {% term integration%} lets you integrate iotty devices into Home Assistant. The iotty family includes the smart switch for lights and gates, the smart shades switch for blinds and shutters, and the smart outlet. 
```
Small tweak to focus the intro more on what the integration does.

> ## Supported devices

Reviewer: ```suggestion

  {% include integrations/config_flow.md %}

## Supported devices
```

- iotty Smart Switch \
  US version
  - https://iottysmarthome.com/products/1-switch-controller?variant=43630747058389
  - https://iottysmarthome.com/products/2-switch-controller?variant=43630751219925
  - https://iottysmarthome.com/products/3-switch-controller?variant=43630760493269
  - https://iottysmarthome.com/products/4-switch-controller?variant=43630774386901 \
  
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli

More integrations are coming soon:

- iotty Shades Switch \
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-shades-switch
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-tende-e-tapparelle

- iotty Smart Outlet: \
  EU VERSION:
  - https://iotty.fr/collections/frontpage/products/prise-intelligent-iotty
  - https://iotty.de/collections/frontpage/products/iotty-smart-steckdose
  - https://iotty.it/collections/frontpage/products/iotty-plus-presa-intelligente

>   

Reviewer: ```suggestion
```

>   {% include integrations/config_flow.md %}

Reviewer: ```suggestion
```

---
title: iotty
description: "Instructions on how to connect iotty Smart Devices to Home Assistant."
ha_release: "2024.8"
ha_category: 
  - Switch
ha_iot_class: "Cloud Polling"
ha_quality_scale: no score
ha_config_flow: true
ha_codeowners:
  - '@pburgio'
ha_domain: iotty
ha_platforms:
  - switch
ha_integration_type: hub
---

The iotty {% term integration%} lets you integrate iotty devices into Home Assistant. The iotty family includes the smart switch for lights and gates, the smart shades switch for blinds and shutters, and the smart outlet. 

> 

Reviewer: Are there any Prerequisites? do I need an account and credentials before I can configure the integration? If so, describe how I get them.

>   {% include integrations/config_flow.md %}

Reviewer: ```suggestion
{% include integrations/config_flow.md %}
```

## Supported devices

This integration currently supports the following iotty Devices:

- iotty Smart Switch \
  US version
  - https://iottysmarthome.com/products/1-switch-controller?variant=43630747058389
  - https://iottysmarthome.com/products/2-switch-controller?variant=43630751219925
  - https://iottysmarthome.com/products/3-switch-controller?variant=43630760493269
  - https://iottysmarthome.com/products/4-switch-controller?variant=43630774386901 \
  
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch
  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli

> More integrations are coming soon:

Reviewer: ```suggestion
```

Reviewer 2: Done

- iotty Shades Switch \
  EU VERSION:
  - https://iotty.uk/collections/frontpage/products/iotty-smart-shades-switch
  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-tende-e-tapparelle

- iotty Smart Outlet: \
  EU VERSION:
  - https://iotty.fr/collections/frontpage/products/prise-intelligent-iotty
  - https://iotty.de/collections/frontpage/products/iotty-smart-steckdose

>   - https://iotty.it/collections/frontpage/products/iotty-plus-presa-intelligente

Reviewer: ```suggestion
```
I think the documentation should focus on the current status.

Reviewer 2: I agree, it has been removed

>   - https://iotty.it/collections/frontpage/products/iotty-plus-presa-intelligente

Reviewer: Please add a section that gives more information about the supported entities. For example, Husquvarna or La Marzocco have sections on switches, for reference.

Reviewer 2: In our integration things are pretty simple: one switch per device. I have added a small section about that, let me know if that is what you were thinking about
